src/desktop/components/react/stitch_components/index.tsx

```diff
@@ -19,7 +19,7 @@ export {
   TwoFactorAuthenticationQueryRenderer as TwoFactorAuthentication,
 } from "v2/Components/UserSettings/TwoFactorAuthentication"
 export { UserEmailPreferencesQueryRenderer as UserEmailPreferences, } from "v2/Components/UserSettings/UserEmailPreferences"
-export { UserInformationQueryRenderer as UserInformation } from "v2/Apps/Settings/Routes/EditProfile/UserInformation"
+export { UserInformationQueryRenderer as UserInformation } from "v2/Components/UserSettings/UserInformation"
```

Comment:
This goes back to what it was before. For any future work let's just leave these code alone and consider it dead. Copy things rather than move them.